pages/api/payment-received.ts

```diff
@@ -31,6 +32,8 @@ const paymentReceivedHandler: NextApiHandler = async (req, res) => {
     const rawApplicationId = req.body.line_items[0]?.properties[0]?.value;
     const shopifyOrderId = req.body.id;
     const applicationId = parseInt(rawApplicationId);
+    const email = req.body.email;
```

Comment:
This is the email inputted in the Shopify checkout (image below).  It's possible for a user to input his phone number instead which won't make it possible to send our own confirmation email. We can toggle a setting in shopify admin to make email mandatory in the check out to avoid that from happening ([link](https://community.shopify.com/c/payments-shipping-and/adding-a-mandatory-phone-number-on-checkout/td-p/603810#:~:text=To%20navigate%20here%20go%20to,can%20only%20checkout%20with%20email.))

![image](https://user-images.githubusercontent.com/51224641/152694987-3f7958f2-bd7b-4734-bea4-c1728e97326d.png)